department-of-veterans-affairs/va.gov-cms
# VAGOV-2836 Update DST

## What this does
 - Synchronizes https://docs.google.com/spreadsheets/d/1vL8rqLqcEVfESnJJK_GWQ7nf3BPe4SSevYYblisBTOI/edit#gid=624373408 with drupal-spect-tool behat tests

## Screenshot
![Screenshot_2019-08-18_11-21-37](https://user-images.githubusercontent.com/2404547/63226668-a3e7f680-c1aa-11e9-83ac-3ab409b0687e.png)

## QA
 - From root directory (or tests), run `lando behat --tags=dst` and visually verify that all tests are green / passing
